text/0000-cargo-install.md

```diff
@@ -0,0 +1,271 @@
+                            package in a repository/path to install.
+    -v, --verbose           Use verbose output
+
+This command manages Cargo's local set of install binary crates. Only packages
+which have [[bin]] targets can be installed, and all binaries are installed into
+`$HOME/.cargo/bin` by default (or `$CARGO_HOME/bin` if you change the home
+directory).
+
+There are multiple methods of installing a new crate onto the system. The
+`cargo install` command with no arguments will install the current crate (as
+specifed by the current directory). Otherwise the `-p`, `--package`, `--git`,
+and `--path` options all specify the source from which a crate is being
+installed. The `-p` and `--package` options will download crates from crates.io.
+
+Crates from crates.io can optionally specify the version they wish to install
+via the `--vers` flags, and similarly packages from git repositories can
+optionally specify the branch, tag, or revision that should be installed. If a
+crate has multiple binaries, the `--bin` argument can selectively install only
+one of them, and if you'd rather install examples the `--example` argument can
+be used as well.
+
+The `--list` option will list all installed packages (and their versions). The
+`--update` option will update either the crate specified or all installed
+crates.
+```
+
+## Installing Crates
+
+Cargo attempts to be as flexible as possible in terms of installing crates from
+various locations and specifying what should be installed. All binaries will be
+stored in the **cargo-local** directory `$CARGO_HOME/bin`. This is typically
+`$HOME/.cargo/bin` but the home directory can be modified via the `$CARGO_HOME`
+environment variable.
+
+Cargo will not attempt to install binaries or crates into system directories
+(e.g. `/usr`) as that responsibility is intended for system package managers.
+
+To use installed crates one just needs to add the binary path to their `PATH`
+environment variable. This will be recommended when `cargo install` is run if
+`PATH` does not already look like it's configured.
+
+#### Crate Sources
+
+The `cargo install` command will be able to install crates from any source that
+Cargo already understands. For example it will start off being able to install
+from crates.io, git repositories, and local paths. Like with normal
+dependencies, downloads from crates.io can specify a version, git repositories
+can specify branches, tags, or revisions.
+
+#### Sources with multiple crates
+
+Sources like git repositories and paths can have multiple crates inside them,
+and Cargo needs a way to figure out which one is being installed. If there is
+more than one crate in a repo (or path), then Cargo will apply the following
+heuristics to select a crate, in order:
+
+1. If the `-p` argument is specified, use that crate.
+2. If only one crate has binaries, use that crate.
+3. If only one crate has examples, use that crate.
```

Comment:
I added this because I figured it'd be a good idea to add `--example` as an option to `cargo install`, but it's definitely non-critical and I'd be fine removing this as well.